Common/Data/Custom/Quandl.cs

```diff
@@ -137,7 +137,7 @@ public override BaseData Reader(SubscriptionDataConfig config, string line, Date
         /// <returns>STRING API Url for Quandl.</returns>
         public override SubscriptionDataSource GetSource(SubscriptionDataConfig config, DateTime date, bool isLiveMode)
         {
-            var source = @"https://www.quandl.com/api/v3/datasets/" + config.Symbol + ".csv?order=asc&api_key=" + _authCode;
+            var source = @"https://www.quandl.com/api/v3/datasets/" + config.Symbol.Value + ".csv?order=asc&api_key=" + _authCode;
```

Comment:
👍   -- if we haven't already -- we should confirm that the mapped ticker is getting pushed into the `config.Symbol.Value` property.